config/locales/forms/business_type_forms/en.yml

```diff
@@ -1,7 +1,15 @@
 en:
   business_type_forms:
     new:
-      heading: Business type
+      heading: What type of business or organisation are you?
+      option_local_authority: Local authority or public body
+      option_limited_company: Limited company (eg a registered company or plc)
+      option_limited_liability_partnership: Limited liability partnership
+      option_individual: Individual (eg a householder)
```

Comment:
Small change. The content from this form came from FRAE which does deal with actual members of the public e.g. I have a house on the river and I want to make a change to the riverbank.

In WCR we don't ever deal with members of the public. So this needs to change to

```yaml
option_individual: Individual (eg a sole trader)
```

Reply:
Oops, good point. Should be fixed now.